reference/supervisor.md

```diff
@@ -767,9 +745,40 @@ indicates the success of the save operation. Be aware that this function can
 overwrite silently existing files, so that the corresponding data may be lost.
 
 > **Note** [C++, Java, Python, Matlab]:
-In the other APIs, the `Robot.saveWorld()` function can be called without
+In the other APIs, the `Robot.worldSave()` function can be called without
 argument. In this case, a simple save operation is performed.
 
+The `wb_supervisor_world_reload()` function sends a request to the simulator
+process, asking it to reload the current world immediately. As a result of
+reloading the current world, the supervisor process and all the robot processes
+are terminated and restarted. You may wish to save some data in a file from your
+supervisor program in order to reload it when the supervisor controller
+restarts.
+
+---
+
+**Name**
+
+**wb\_supervisor\_simulation\_reset** - *reset the simulation*
+
+{[C++](cpp-api.md#cpp_supervisor)}, {[Java](java-api.md#java_supervisor)}, {[Python](python-api.md#python_supervisor)}, {[Matlab](matlab-api.md#matlab_supervisor)}, {[ROS](ros-api.md)}
+
+```c
+#include <webots/supervisor.h>
+
+void wb_supervisor_simulation_reset();
+```
+
+**Description**
+
+The `wb_supervisor_simulation_reset()` function sends a request to the simulator process, asking it to reset the simulation immediately.
+The simulation is not entirely destructed and reconstructed like when calling `wb_supervisor_world_reload`, but the initial state of all the nodes is restored, which is way faster.
```

Comment:
For consistency with the new changes of @fabienrohrer  you should use the function suffix:
" when calling `wb_supervisor_world_reload`" -> " when calling the `wb_supervisor_world_reload` function"